css-three-column-layout/styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 100%;
+  padding: 100px 0;
+  margin: 0 auto;
+  background-color: #1a96da;
+}
+
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.column-full {
+  max-width: 100%;
+  text-align: center;
+  margin: auto;
+}
+
+.column-one-third {
+  text-align: center;
+  width: calc(100% / 3);
+  padding: 10px 70px;
+}
```

Comment:
The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.